index.html

```diff
@@ -169,10 +169,26 @@ <h5 class="media-text">LET’S CONNECT</h5>
                     <img src="./assets/Normal Button/Tertiary/Icons/Icon - Chevron - Right - Larger.svg">
                 </li>
                 <div class="line"></div>
-            </ul>
+
+            </div>
         </div>
 
     </section>
+    <section class="contact-section">
+        <h2 class="contact-heading">Contact me</h2>
+        <p class="contact-cta">If you have an application you are
+            interested in developing, a feature
+            that you need built or a project that
+            needs coding. I’d love to help with it </p>
+        <form action="#" class="contact-form">
```

Comment:
- [ ] Your form is missing the action and method attribute to integrate with [formspree service](https://formspree.io/html). Kindly follow the 'get started' section and make sure you have a formspree action link so the form can be submitted.